src/components/Home.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable max-len */
+const Home = () => (
+  <div className="home">
+    <h2>Welcome to my page!</h2>
+    <p>
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque quae quis laborum hic! Impedit ea suscipit incidunt veniam voluptatem perspiciatis ipsa voluptatibus nisi pariatur eius quam corporis esse nobis aspernatur, debitis illum aperiam
+      totam eveniet, minus sit ab. Quas officia consectetur temporibus, praesentium culpa iusto tempore laudantium? Provident laboriosam quos, quibusdam numquam, sed placeat officiis, fugiat architecto nihil iure ex.
+    </p>
+    <p>
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque quae quis laborum hic! Impedit ea suscipit incidunt veniam voluptatem perspiciatis ipsa voluptatibus nisi pariatur eius quam corporis esse nobis aspernatur, debitis illum aperiam
+      totam eveniet, minus sit ab. Quas officia consectetur temporibus, praesentium culpa iusto tempore laudantium? Provident laboriosam quos, quibusdam numquam, sed placeat officiis, fugiat architecto nihil iure ex.
+    </p>
+  </div>
```

Comment:
While you did an amazing job working on the project requirements I noticed you disabled eslint in this given file. It is against microverse policy to disable linters because linters help us write clean code it is also not good practice to disable linters so it would be nice if you enabled eslint in this file.